test/clojure_tictactoe/game/board_test.clj

```diff
@@ -0,0 +1,39 @@
+(ns clojure-tictactoe.game.board-test
+  (:require [clojure.test :refer :all]
+            [clojure.string :as string]
+            [clojure-tictactoe.helpers :as helper]
+            [clojure-tictactoe.game.board :refer :all]))
+
+
+
+(def empty-board
+  [0 1 2 3 4 5 6 7 8])
+
+(def x-on-one-board
+  [0 "x" 2 3 4 5 6 7 8])
+
+(testing "Board"
+
+  (testing "space is open"
+    
+    (deftest open?-test-is-open
+      (testing "open? returns true when no marker on space"
+        (is (true? (open? 1 empty-board)))))
+    
+    (deftest open?-test-is-not-open
+      (testing "open? returns false when marker is on space"
+        (is (false? (open? 1 x-on-one-board)))))
+
+   (deftest open?-test-integer-string
```

Comment:
Indent is off by one space here.

Tip: in vim, if you highlight a block of text in visual mode and then press `=`, it will automatically fix the indents for you.

Reply:
Yeah, I just went looking for a way to mark all whitespace chars because I am having a hard time seeing in vim when I need to use that command. Is there a better option than https://github.com/nathanaelkane/vim-indent-guides ? I want all whitespace, but everything looks like it's for leading or trailing.

Reply:
Hm, I haven't tried it myself, but what about this: https://stackoverflow.com/questions/1675688/make-vim-show-all-white-spaces-as-a-character

Alternatively, I think if you just type: `/\s\+` in normal mode and press enter, it will highlight all whitespace for you. Of course, if you search for something else, it will then highlight that new search term instead.